pkg/cri/sbserver/instrumented_service.go

```diff
@@ -948,6 +952,8 @@ func (in *instrumentedAlphaService) PullImage(ctx context.Context, r *runtime_al
 	if err := in.checkInitialized(); err != nil {
 		return nil, err
 	}
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("PullImage"))
```

Comment:
Should we make this `strings.Join` a part of tracing package API?

For instance, `StartSpan` could accept arbitrary number of strings, like this:
`tracing.StartSpan(ctx, criSbServerSpanPrefix, "PullImage")`

Reply:
We can but does it bring any value? it will be simpler if the instrumentation side makes the decision on how to create the span name( add prefix, suffix or use any custom logic)

Reply:
>  add prefix, suffix or use any custom logic

Does it make sense to be consistent everywhere?
Like we'd specify `tracing.StartSpan(ctx, "a", "b", "c")` and it'll apply same separator across the project.

Reply:
Okay, I see your point. I am planning to change the current span methods once I add a wrapper object, so I will add this change in the follow up PR.